src/models/cloudProviderIdentifier.ts

```diff
@@ -13,13 +13,26 @@ export enum CloudAdapterIdentifier {
 }
 
 export enum CloudProviderIdentifier {
-    GENEZIO = "genezio",
+    GENEZIO_CLOUD = "genezio-cloud",
+    GENEZIO_UNIKERNEL = "genezio-unikernel",
+    GENEZIO_CLUSTER = "genezio-cluster",
+    GENEZIO_AWS = "genezio-aws",
+
     SELF_HOSTED_AWS = "selfHostedAws",
```

Comment:
MAYBE: rename this to `SELF_HOSTED_AWS_LEGACY` and add `SELF_HOSTED_AWS = "self-hosted-aws"`

Reply:
Done.